lib/metrics.py

```diff
@@ -30,14 +30,18 @@ class MetricsServerEmitter(MetricsEmitter):
     def __init__(self, metrics_url):
         self.metrics_url = metrics_url
         self.fallback_emitter = LoggingEmitter()
+        self.logfailure = True
 
     def emit(self, stats):
         try:
             response = requests.post(self.metrics_url, json=stats, timeout=10)
+            self.logfailure = True
         except Exception as e:
-            logger.debug(
-                "Failed to send metrics to trends server.", exc_info=True
-            )
+            if self.logfailure:
+                logger.warning(
```

Comment:
We already did DEP-511 here: https://github.com/mendix/cf-mendix-buildpack/pull/189, so the changes to the logging are not needed.